packages/firebase/index.d.ts

```diff
@@ -4076,6 +4090,20 @@ declare namespace firebase.analytics {
       },
       options?: firebase.analytics.AnalyticsCallOptions
     ): void;
+               
+     /**
+     * Sends analytics event with given `eventParams`. This method
+     * automatically associates this logged event with this Firebase web
+     * app instance on this device.
+     * List of official event parameters can be found in
+     * {@link https://developers.google.com/gtagjs/reference/event
+     * the gtag.js reference documentation}.
+     */
+    logEvent(
+      eventName: 'page_view',
+      eventParams?: {},
```

Comment:
For `page_view` I think it should be:

```
eventParams: {
        page_title?: string;
        page_location?: string;
        page_path?: string;
        [key: string]: any;
      }
```
Source: https://developers.google.com/analytics/devguides/collection/gtagjs/pages